R/generateEDGEdata.R

```diff
@@ -29,13 +30,14 @@
 #' @export
 
 
-toolGenerateEDGEdata <- function(input_folder, output_folder, cache_folder = NULL,
-                                 SSP_scen = "SSP2", tech_scen = "Mix", smartlifestyle = FALSE,
-                                 storeRDS = FALSE,
-                                 gdxPath = NULL,
-                                 preftab = NULL, plot.report = FALSE,
-                                 mitab4W.path = NULL, mitab.path = NULL,
-                                 ssp_demreg.path = NULL, regional_demreg.path = NULL, FEPricetab = NULL) {
+generateEDGEdata <- function(input_folder, output_folder, cache_folder = NULL,
```

Comment:
is there a good reason to rename this function? the tool prefix helps to have it monitored by `mrremind`...

Reply:
Well spotted, that was not my intention